libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -111,292 +107,9 @@ function getDescriptionForDynamicallyAddedParameterType(type: DynamicallyAddedPa
-      const [schemaKey, propertyValue] = Object.entries(nextProperty)[0];
-      return { ...resultPropertiesObj, [schemaKey]: propertyValue };
-    }, {});
-
-  let rootObject: Schema;
-
-  rootObject = {
-    type: 'object',
-    properties,
-    required: requiredArray,
-  };
-
-  if (isRequestApiConnectionTrigger) {
-    rootObject = {
-      rows: {
-        type: 'array',
-        items: {
-          type: 'object',
-          properties,
-          required: requiredArray,
-        },
-      },
-    };
-  }
-
-  return [
-    {
-      id: guid(),
-      type: ValueSegmentType.LITERAL,
-      value: JSON.stringify(rootObject),
-    },
-  ];
-}
-
-export function getEmptySchemaValueSegmentForInitialization(useStaticInputs: boolean, isRequestApiConnectionTrigger = false) {
-  let rootObject: Schema;
-
-  rootObject = {
-    type: 'object',
-    properties: {},
-    required: [],
-  };
-
-  if (isRequestApiConnectionTrigger) {
-    rootObject = {
-      rows: {
-        type: 'array',
-        items: {
-          type: 'object',
-          properties: {},
-          required: [],
-        },
-      },
-    };
-  }
-
-  if (useStaticInputs) {
-    rootObject = rootObjectWithStaticInputs;
-  }
-
-  return [
-    {
-      id: guid(),
-      type: ValueSegmentType.LITERAL,
-      value: JSON.stringify(rootObject),
-    },
-  ];
-}
-
-export function createDynamicallyAddedParameterProperties(
-  itemType: DynamicallyAddedParameterTypeType,
-  schemaKey: string
-): IDynamicallyAddedParameterProperties {
-  let format, fileProperties;
-  let type = '';
-  switch (itemType) {
-    case DynamicallyAddedParameterType.Date:
-    case DynamicallyAddedParameterType.Email:
-      type = constants.SWAGGER.TYPE.STRING;
-      format = itemType.toLowerCase();
-      break;
-    case DynamicallyAddedParameterType.Text:
-      type = constants.SWAGGER.TYPE.STRING;
-      break;
-    case DynamicallyAddedParameterType.File:
-      type = constants.SWAGGER.TYPE.OBJECT;
-      fileProperties = {
-        contentBytes: { type: constants.SWAGGER.TYPE.STRING, format: constants.SWAGGER.FORMAT.BYTE },
-        name: { type: constants.SWAGGER.TYPE.STRING },
-      };
-      break;
-    case DynamicallyAddedParameterType.Boolean:
-      type = constants.SWAGGER.TYPE.BOOLEAN;
-      break;
-    case DynamicallyAddedParameterType.Number:
-      type = constants.SWAGGER.TYPE.NUMBER;
-      break;
-  }
-
-  return {
-    description: getDescriptionForDynamicallyAddedParameterType(itemType),
-    format,
-    title: convertDynamicallyAddedSchemaKeyToTitle(schemaKey, itemType),
-    type,
-    properties: fileProperties,
-    'x-ms-content-hint': itemType,
-    'x-ms-dynamically-added': true,
-  };
-}
-
-export function generateDynamicParameterKey(
-  parameters: DynamicallyAddedParameterProps[],
-  typeHint: DynamicallyAddedParameterTypeType
-): string {
-  const currentKeys = parameters.map((parameter) => parameter.schemaKey);
```

Comment:
What's the purpose of removing this map?  Can we make it 
`const currentKeys = parameters.map((parameter) => parameter?.schemaKey ?? parameter);`